tests/index.js

```diff
@@ -1,7 +1,8 @@
-var tape       = require('tape')
-  , workerFarm = require('../')
-  , childPath  = require.resolve('./child')
-  , fs         = require('fs')
+var tape          = require('tape')
+  , child_process = require('child_process')
```

Comment:
@slang800 Please, this issue does not need more pedantry. Code being a work of art is irrelevant if it doesn't work.